30414/Dolineanu A. Vlad Ionut/Assignment 2/functions.h

```diff
@@ -0,0 +1,37 @@
+#ifndef FUNCTIONS_H_INCLUDED
+#define FUNCTIONS_H_INCLUDED
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define ALLOC_NODE (node*)malloc(sizeof(node))
+
```

Comment:
Is this define something I should be doing? I find it helps quite a lot but I don't know if it has other side effects